Pages/Authentication/login.css

```diff
@@ -0,0 +1,42 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital@1&family=Ubuntu:wght@300&display=swap');
+body {
+    font-family: 'Source Sans Pro', sans-serif;
+    font-family: 'Ubuntu', sans-serif;
+    margin: 0px;
+}
+
+:root {
+    --primary-color: #2563EB;
+    --gold: rgb(253, 196, 52);
```

Comment:
Please use hex code for consistency.

Reply:
Can you help me, How can we get hex code of a rgb color.

Reply:
if you click up while choosing rgb color you can change with hex code insteead rgb